baselines/ppo1/pposgd_simple.py

```diff
@@ -3,20 +3,23 @@
 import baselines.common.tf_util as U
 import os
 import tensorflow as tf, numpy as np
+from tensorflow.python.framework import graph_util, graph_io
 import time
 from baselines.common.mpi_adam import MpiAdam
 from baselines.common.mpi_moments import mpi_moments
 from mpi4py import MPI
 from collections import deque
 
-def traj_segment_generator(pi, env, horizon, stochastic):
+def traj_segment_generator(pi, env, horizon, stochastic, rw_scaler):
```

Comment:
The Reward Scaler is an idea I found in a paper, where the reward is normalized using running statistics (just like the way inputs are normalized). It does not worked much better - actually, just using a multiplication factor was better. But I maintained here because in some problems it can be more helpful.

Reply:
which paper was it? Do you remember? I think I've saw something related in a deepmind paper